bin/refmt

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env node
+"use strict";
+
+var child_process = require('child_process')
+
+var exe = __filename + ".exe"
+var delegate_args = process.argv.slice(2)
+
+try {
+    child_process.execFileSync(exe, delegate_args, { stdio: 'inherit' })
+} catch (e) {
```

Comment:
is there any value in surfacing what the error was, even just as far as the message?